odoo/tools/misc.py

```diff
@@ -1132,6 +1132,15 @@ def html_escape(text):
     def html_escape(text):
         return werkzeug.utils.escape(text)
 
+def babel_locale_parse(lang_code):
+    try:
+        return babel.Locale.parse(lang_code)
+    except:
```

Comment:
small reminder: in normal business code we always want `except` clauses to catch at least `Exception`, [otherwise they'll also catch `SystemExit` and `KeyboardInterrupt`](https://docs.python.org/3/library/exceptions.html#exception-hierarchy), which can leads to unexpected behavior.

Whenever you can, catching specific exception class is preferred. If you only expect an `UnknownLocaleError`, catch that, because you probably did not meant to catch `MemoryError` or `SyntaxError`, which could hide another problem :)

Reply:
@odony Indeed, sorry, thanks for the reminder. Do I create a PR to change it?

Reply:
> @odony Indeed, sorry, thanks for the reminder. Do I create a PR to change it?

yes (and if a foward-port was open we  could fix it in the forward-port and have new PR only up to that forward-port non included)